python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-ospf.j2

```diff
@@ -200,6 +200,18 @@ router ospf {{ process_id.id }}
    summary-address {{ summary_address.prefix }}
 {%         endif %}
 {%     endfor %}
+{%     if process_id.graceful_restart.enabled is arista.avd.defined(true) %}
+{%         set graceful_restart_cli = "graceful-restart" %}
+{%         if process_id.graceful_restart.grace_period is arista.avd.defined %}
+{%             set graceful_restart_cli = graceful_restart_cli ~ " grace-period " ~ process_id.graceful_restart.grace_period %}
+{%         endif %}
+   {{ graceful_restart_cli }}
+{%     endif %}
+{%     if process_id.graceful_restart_helper is arista.avd.defined(true) %}
+   graceful-restart-helper
```

Comment:
wan2-dc1(config-router-ospf)#sh ac
router ospf 11
   max-lsa 12000
   summary-address 10.1.1.1/32
   graceful-restart grace-period 10
   mpls ldp sync default
   no graceful-restart-helper
   
   seems config is not in correct order

Reply:
Fixed